### PR TITLE
Put fabric8 mvn tooling in fabric8 profile

### LINKED ISSUE #30: Support running outside of kubernetes
For what reason do we involve fmp during the build? I like running

mvn clean install

from the parent module to compile the whole thing but that requires a running k8s. Do we really need that?

### PR DESCRIPTION
**Pull request by jimmidyson**
Fixes #30

### CODE REVIEW

File: runtime/pom.xml [hunk 1]
```diff
@@ -250,4 +237,29 @@
...
+    <profile>
+      <id>fabric8</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>io.fabric8</groupId>
+            <artifactId>fabric8-maven-plugin</artifactId>
+            <version>${fabric8.maven.plugin.version}</version>
+            <executions>
+              <execution>
```
Comment (rhuss) [inline runtime/pom.xml]: I would leave the configuration outside the profile and only remove the executions or move the executions into a profile. That way people can use goals like 'fabric8:resource' without specifying this profile.

Comment (jimmidyson) [inline runtime/pom.xml]: Done. Added executions in `fabric8` profile.

**Comment by rhuss** [comment]
thanks !

lgtm

**Comment by rhuss** [comment]
[merge]

**Pull request merged**